src/components/ContactProfile/ContactProfile.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import ProfileContent from './ProfileContent'
+
+export default function ContactProfile({ contacts }) {
```

Comment:
Do you really need the component ContactProfile? It seems like it doesn't add anything to the ProfileContent component.

Reply:
> I didn't understand why the ProfileContent component has no props. That might be a bug. Otherwise it looks great!

I put the static contact information in the wrong place, so it worked without props. I have fixed it! Thanks for the hint.

Reply:
I want to add a header to the component in further stories. I have prepared the component ContactProfile for this.